system/jlib/jsmartsock.ipp

```diff
@@ -62,19 +62,20 @@ class jlib_decl CSmartSocketFactory: public Thread,
     implements ISmartSocketFactory
 {
     SmartSocketEndpointArray sockArray;
-    Mutex lock;
 
     unsigned nextEndpointIndex;
     bool retry;
     unsigned retryInterval;
-    unsigned dnsInterval;
 
     void shuffleEndpoints();
     SmartSocketEndpoint *findEndpoint(SocketEndpoint &ep);
 
 public:
     IMPLEMENT_IINTERFACE;
 
+    Mutex lock;
+    unsigned dnsInterval;
```

Comment:
See other comment in CSecureSmartSocketFactory::connect_timeout ..
If common code is moved down into base, then these don't need exposing/making public